### ISSUE TITLE
Gulp v4 creating files with corrupt timestamps

### ISSUE THREAD

**Issue by grassick** [gulpjs/gulp#1328]
Something has recently broken with gulp v4. Not only does watchify not work correctly (no longer emits 'end' events), timestamps are also wildly corrupt:

```
-rw-r--r--  1 clayton clayton 1238 Oct 15 11:49 index_map.html
-rw-r--r--  1 clayton clayton 1192 -9223372036854775808 manifest.appcache
-rw-r--r--  1 clayton clayton  196 Feb 20  2015 Readme.md
-rw-r--r--  1 clayton clayton   25 Feb 20  2015 robots.txt
```

Note the negative timestamp which causes all sorts of issues.

The new (broken) dependencies vs old (working) are:
chokidar: 1.2.0 vs 1.2.0
gulp-cli: 0.4.0 vs 0.4.0
undertaker: 0.12.0 vs 0.12.0
vinyl-fs: 2.2.0 vs **2.1.1**

I strongly suspect vinyl-fs. Opening an issue there, but can we downgrade to 2.1.1 in the meantime?

I've confirmed that it solves the issue.

**Comment by grassick** [comment]
Underlying issue: https://github.com/gulpjs/vinyl-fs/issues/113

**Comment by phated** [comment]
This is being worked on in a vinyl-fs PR.  We are just trying to figure out the best way to test the fix.

**Comment by grassick** [comment]
Temporary fork at https://github.com/mWater/gulp#4.0 for those who are unable to build because of this issue. Only change is 2.1.1 for vinyl-fs.

**Comment by phated** [comment]
Sorry for the delay.  I wanted to make sure our tests had a reproduction case and I could verify we fixed this before was published.  I've just published this as 2.2.1 - please let me know how this works out for you.

**Comment by grassick** [comment]
Works great! Thanks for the fast work on this.
